src/routes/boxer.js

```diff
@@ -0,0 +1,57 @@
+const { Boxer } = require('../db');
+
+const createBoxer = async (res,req) =>{
+    const{ boxerName, fightStyle } = req.body;
+    const boxer = Boxer.build({ boxerName, fightStyle });
+    await boxer.save();
+    res.status(200).send(boxer);
+};
+
+const listBoxers = async (req, res) => {
+    const boxers = await Boxer.findAll();
+    res.status(200).send(boxers);
+};
+
+const getBoxer = async (req, res) => {
+    const boxers = await Boxer.findAll({
+        where:{
+            id: req.params.id,
+        },
+    });
+    if(boxer.length > 0) {
+        res.status(200).send(boxers[0]);
+    }else{
+        res.status(400).send(`Could not find boxer with id ${req.params.id}`);
+    }
+};
+
+const deleteBoxer = async (req, res) => {
+    const boxers = await Boxer.destroy({
+        where: {
+            id: req.params.id,
+        },
+    });
+    if(boxer.length > 0) {
+        res.status(200).send('Error');
+    } else{
+        res.status(200).send('Boxer Has Been Deleted');
+    }
+};
+
+const updateBoxer = async (req, res) => {
+    await Boxer.update({boxerName: req.query.boxerName, fightStyle: req.query.fightStyle},{
+        where: {
+            id: req.params.id,
+        },
+        returning: true,
+    });
+    res.status(200).send('updated Boxer');
+};
+
+module.exports = {
+    createBoxer,
+    listBoxers,
+    getBoxer,
+    deleteBoxer,
+    updateBoxer
```

Comment:
Change to `module.exports = router;` because it will lead to the routes and all the functions